src/elements/Image/Image.tsx

```diff
@@ -24,10 +26,17 @@ const getContainerStyle = (node: ImageNode): CSSProperties => {
     return { width };
 };
 
-export const Image: FunctionComponent<Props> = ({ children, className, node, ...props }) => {
+export const Image: FunctionComponent<Props> = ({
+    children,
+    className,
+    node,
+    onDownload,
+    onPreviewOpen,
+    ...props
+}) => {
     const { file, href, layout } = node;
     const [isPreviewOpen, setIsPreviewOpen] = useState<boolean>(false);
-    const image = UploadcareImage.createFromPrezlyStoragePayload(file);
+    const image = useMemo(() => UploadcareImage.createFromPrezlyStoragePayload(file), [file.uuid]);
```

Comment:
The `useMemo` here is required so Lightbox's `useEffect` doesn't trigger infinitely.

Reply:
Hm... Do you think there can be a problem when there are multiple images with the same uuid, but different effects/cropping applied?

Reply:
I don't think it's possible because each image (even if it has the same uuid) would have its own instance of `Image` component and the memoized values are local to the component itself.

I think it would only be a problem if the `file` prop changed after rendering, however we don't have such use case at this moment so I don't think it's an issue.

Reply:
Makes sense. Thanks!